### DISCUSSION TITLE
Contributing

### DISCUSSION THREAD

**Discussion by CodeLeom** [hiero-ledger/hiero#2] [General]
I saw that there are no open issues on the Hiero repository yet and work has not really started. I would like to contribute to the documentation, organize event and move things forward

**Comment by MilanWR** [comment]
I believe the repo still has to be moved over from Hashgraph https://github.com/hashgraph

**Reply by CodeLeom** [reply]
can't wait to start contributing. @MilanWR do you have any idea on anything that can be done from the community side to help with

**Reply by MilanWR** [reply]
I'd say start learning about how to use Hiero/Hedera if you haven't yet! there is great documentation here: docs.hedera.com (maybe following a tutorial? https://docs.hedera.com/hedera/tutorials)

**Comment by hendrikebbers** [comment]
Hi all,
First of all, sorry for the delay in providing an official answer here. What @MilanWR said is 100% correct. When we announced Hiero, all projects were still part of the Hedera GitHub organization at https://github.com/hashgraph. This will change in the near future, and I hope that the first repos (SDKs) will be ready for us to start transferring within the next few days. With that transfer several issues will be ready for contribution. We currently work hard on getting a lot of good "good first issue"s prepared for the launch and for Hacktoberfest. @CodeLeom please give us some more days for preparing all this :)

**Reply by CodeLeom** [reply]
I look forward to getting my first contribution with the project.

**Reply by hendrikebbers** [reply]
I just prepared 28 "good first issue"s for Hacktoberfest :) https://github.com/issues?q=is%3Aopen+is%3Aissue+org%3Ahashgraph+archived%3Afalse+label%3Ahacktoberfest

They are still in the hashgraph org but we started transition and I just created a pr to get all the updates: https://github.com/LFDT-Hiero/hiero/pull/3

**Reply by CodeLeom** [reply]
this is amazing, thank you @hendrikebbers. If I am contributing, the PR will be to the hashgraph org or I should wait for it to be transitioned here?

**Reply by hendrikebbers** [reply]
The PRs (sadly) must be done at hashgraph. We are working on the transfer right now.

**Reply by CodeLeom** [reply]
No problem. Thank you for your support and feedback

**Comment by hendrikebbers** [comment]
https://hiero.org/hacktoberfest/ here we go :)

**Reply by Princesso** [reply]
Thanks for sharing the link @hendrikebbers. I was also looking to contribute and had the same questions as @CodeLeom.

**Comment by hendrikebbers** [comment]
:octocat: First Hacktoberfest contributions merged :tada: and the first new people are already part of our Hiero Heros page: https://hiero.org/heros/
